docs/content/3.customization.md

```diff
@@ -61,6 +61,28 @@ export default defineAppConfig({
 })
 ```
 
+### Contact us section
+
+Toolkit has the set of default contacts. 
+If you need to use project-specific contacts, add items with a following syntax to `contacts` array at `app.config.ts` file of your docs website:
```

Comment:
User will override all contacts in this case. It is not clear from the text.

Reply:
At line 82 of this document I've added follownig notice:
```
::notice{info}
Default contacts will be hidden if you add any project-specific ones
::
```

Isn't it enough?

Reply:
In this case I would just think about word "add". Because it seems misleading for me. My first thought was I will add new contacts

Reply:
```suggestion
If you need to use project-specific contacts, specify items with a following syntax to `contacts` array at `app.config.ts` file of your docs website:
```